auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -492,6 +522,37 @@ export default class EntityManagement extends Mixins(AccountChangeMixin, NextPag
   border-color: rgb(33, 33, 33); // grey darken-4
 }
 
+.v-tooltip__content {
+  background-color: RGBA(73, 80, 87, 0.95) !important;
+  color: white !important;
+  border-radius: 4px;
+  font-size: 12px !important;
+  line-height: 18px !important;
+  padding: 15px !important;
+  letter-spacing: 0;
+  max-width: 360px !important;
+}
+
+.v-tooltip__content:after {
+  content: "" !important;
+  position: absolute !important;
+  top: 50% !important;
+  right: 100% !important;
+  margin-top: -10px !important;
+  border-top: 10px solid transparent !important;
+  border-bottom: 10px solid transparent !important;
+  border-right: 8px solid RGBA(73, 80, 87, .95) !important;
+}
+
+.top-tooltip:after {
+  top: 100% !important;
+  left: 45% !important;
+  margin-top: 0 !important;
+  border-right: 10px solid transparent !important;
+  border-left: 10px solid transparent !important;
+  border-top: 8px solid RGBA(73, 80, 87, 0.95) !important;
+}
+
```

Comment:
Let's keep an eye on all this CSS. We might want to make it global (by adding to base.scss) if it applies to all tooltips, which it probably should but we may not be ready for that (needs UI/UX input).

Reply:
I agree, I think there are 6 separate instances where this or similar code is being used. I think the biggest problem is the arrows and if the tooltip appears on the side or the top ect

Reply:
Well, some tooltips are "top-tooltip" and others are "bottom-tooltip" so they'll have different classes.

Reply:
Could the CSS be made global now (easily)?

Reply:
I don't think so there isn't consistency across the css, this maybe should be made into a tech debt ticket.